### PR TITLE
feat: add `solid` support

### PR DESCRIPTION
**Pull request by shlroland**

### CODE REVIEW

File: packages/react/package.json [hunk 1]
```diff
@@ -13,8 +13,8 @@
     "react"
   ],
   "sideEffects": false,
-  "main": "./lib/index.es.js",
-  "types": "./lib/index.d.ts",
+  "main": "./src/index.ts",
```
Comment (Saul-Mirone) [inline packages/react/package.json:16]: This change will make the library can't be used without a bundler. Why we need this change?

**Review by Saul-Mirone — CHANGES_REQUESTED** [review]

File: packages/react/package.json (hunk 1 above)
Comment (shlroland) [reply packages/react/package.json:16]: Apologies, the modifications mentioned above were made during my local testing and I forgot to revert them before committing. I will make sure to refine them along with the other sections.

**Comment by shlroland** [comment]
@Saul-Mirone Done. I added a readme file by referring to other packages. 

However, when running the cypress test files on my computer, I encountered errors. Currently, I have not included any test cases specifically for solid.

**Comment by Saul-Mirone** [comment]
Are you using corepack? It seems that there's a mismatch in the lock file.

**Comment by shlroland** [comment]
Hi, @Saul-Mirone. Is there anything else that needs to be modified?

**Comment by Saul-Mirone** [comment]
> Hi, @Saul-Mirone. Is there anything else that needs to be modified?

I'll try the feature on my local environment today and merge the pr if everything works fine.

**Pull request merged**